### PR TITLE
Updated readme where we dropped zfcBase for zfcUser 2.x branch

### PR DESCRIPTION
**Pull request by stijnhau**
Updated readme where we dropped zfcBase for zfcUser 2.x branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,19 +65,36 @@ Installation
...
     "require": {
         "zf-commons/zfc-user": "dev-master"
     }
-    ```
+    ```                         |
+|-----------------|--------------------------------------------------------------------------------|
+| 2.x             | 1. Add this project in your composer.json:                                     |
+                        ```json                                                                    |
+                        "require": {                                                               |
+                            "zf-commons/zfc-user": "dev-master"                                    |
```
Comment (Danielss89) [inline README.md]: Should probably not be dev-master anymore?

Comment (stijnhau) [reply README.md]: Thats true changed it now

Comment (stijnhau) [reply README.md]: Can trhis be merged?

**Comment by stijnhau** [comment]
Can this be merged in?

**Pull request merged**